frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/menu_user.dart

```diff
@@ -54,6 +60,49 @@ class MenuUser extends StatelessWidget {
     );
   }
 
+  Widget _renderThemeToggle(BuildContext context) {
+    final themeProvider = Provider.of<ThemeProvider>(context);
+    final provider = Provider.of<ThemeProvider>(context, listen: false);
+    final theme = context.watch<AppTheme>();
+    return Switch.adaptive(
+      value: themeProvider.isDarkMode,
+      onChanged: (value) {
+        final provider = Provider.of<ThemeProvider>(context, listen: false);
+        provider.toggleTheme(value);
+        print(themeProvider.isDarkMode);
+      },
+    );
+    // return Material(
```

Comment:
Comment out code should not be submitted

Reply:
Will submit the final edits after seeing which code works better.